Sources/Orbit/Components/Stepper.swift

```diff
@@ -25,8 +25,14 @@ public struct Stepper: View {
     }
     
     @ViewBuilder var valueText: some View {
-        Text("\(value)")
-            .accessibility(.stepperValue)
+        if #available(iOS 14, *) {
```

Comment:
We should be able to use the iOS13 version `accessibility(value: Text)` as we do it for other components.

Reply:
You're right, it works with that, too. Replaced.